core/qtserialbluetooth.cpp

```diff
@@ -284,10 +284,18 @@ ble_packet_open(dc_iostream_t **iostream, dc_context_t *context, const char* dev
 
 	static const dc_custom_cbs_t callbacks = {
 		.set_timeout	= qt_ble_set_timeout,
+		.set_break	= nullptr,
```

Comment:
Another way of solving this might be to use the 'fieldname: value' syntax https://gcc.gnu.org/onlinedocs/gcc/Designated-Inits.html

Although that has apparently been obsolete since gcc 2.5. I only came across it after googling the lack of support in g++ for the dot syntax.

Reply:
Yes. I looked at that. It creates rather verbose warnings with clang which tell you that you are using a non-standard gcc language extension and helpfully shows you the standards compliant correct way to do this.
So yeah, this one is on the gcc community, but since we still use the older compilers on many platforms, it will be a few years before there is a clean solution. For now I think I'll merge this as is and do a 4.9.6 release over the weekend.